text/0006-dynamic-pricing-for-bulk-coretime-sales.md

```diff
@@ -0,0 +1,189 @@
+# RFC-0006: Dynamic Pricing for Bulk Coretime Sales
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | July 09, 2023                                                                               |
+| **Description** | A dynamic pricing model to adapt the regular price for bulk coretime sales                  |
+| **Authors**     | Tommi Enenkel (Alice und Bob)                                                               |
+| **License**     | MIT                                                                                         |
+
+## Summary
+
+This RFC proposes a dynamic pricing model for the sale of Bulk Coretime on the Polkadot UC. The proposed model updates the regular price of cores for each sale period, by taking into account the number of cores sold in the previous sale, as well as a limit of cores and a target number of cores sold. It ensures a minimum price and limits price growth to a maximum price increase factor, while also giving govenance control over the steepness of the price change curve. It allows governance to address challenges arising from changing market conditions and should offer predictable and controlled price adjustments.
+
+## Motivation
+
+RFC-1 proposes periodic Bulk Coretime Sales as a mechanism to sell continouos regions of blockspace (suggested to be 4 weeks in length). A number of Blockspace Regions (compare RFC-1 & RFC-3) are provided for sale to the Broker-Chain each period and shall be sold in a way that provides value-capture for the Polkadot network. The exact pricing mechanism is out of scope for RFC-1 and shall be provided by this RFC. 
```

Comment:
RFC-3 is a companion to RFC-1, for underlying improvements to scheduling infrastructure - the goal is for them to communicate via RFC-5. But it's probably out of scope for anything related to pricing/sales

Reply:
was that just a comment on the relationship of the RFCs or should I change something?